packages/sitecore-jss-nextjs/src/utils.ts

```diff
@@ -75,3 +76,9 @@ export const getJssEditingSecret = (): string => {
   }
   return secret;
 };
+
+export const resolvePointOfSale = (site: SiteInfo, language: string): string => {
```

Comment:
Should there be a test case if empty string is passed for `language` to the `resolvePointOfSale`?